website/docs/docs/build/sl-getting-started.md

```diff
@@ -14,38 +14,34 @@ import DefineMetrics from '/snippets/_sl-define-metrics.md';
 import ConfigMetric from '/snippets/_sl-configure-metricflow.md';
 import TestQuery from '/snippets/_sl-test-and-query-metrics.md';
 
-This getting started page presents a sample workflow to help you create your first metrics. It uses the [Jaffle shop example project](https://github.com/dbt-labs/jaffle-sl-template) as the project data source and is available for you to use. If you prefer, you can create semantic models and metrics for your own dbt project.
+This getting started page presents a sample workflow to help you create your first metrics in dbt Cloud or the command line (CLI). It uses the [Jaffle shop example project](https://github.com/dbt-labs/jaffle-sl-template) as the project data source and is available for you to use. If you prefer, you can create semantic models and metrics for your own dbt project.
 
-To fully experience the power of a universal dbt Semantic Layer, take the following steps:
+This guide will cover the following sections and steps:
 
-- [Create a semantic model](#create-a-semantic-model) with MetricFlow
-- [Define metrics](#define-metrics) with MetricFlow
-- [Test and query metrics locally](#test-and-query-metrics) with MetricFlow
+- [Create a semantic model](#create-a-semantic-model) using MetricFlow
+- [Define metrics](#define-metrics) using MetricFlow
+- [Test and query metrics locally](#test-and-query-metrics) using MetricFlow
 - [Run a production job](#run-a-production-job) in dbt Cloud
 - [Set up dbt Semantic Layer](#set-up-dbt-semantic-layer) in dbt Cloud
 - [Connect to and query the API](#connect-and-query-api) with dbt Cloud
 
+To fully experience the power of a universal [dbt Semantic Layer](/docs/use-dbt-semantic-layer/dbt-sl), you'll need a dbt Cloud [Team or Enterprise](https://www.getdbt.com/pricing/) account. 
```

Comment:
This line was a little weird to me - it indicates that they can use the semantic layer without cloud, they just won't get the full experience. The first question I'd ask is "what's available in Core and what is in Cloud?"
```suggestion
To experience the power of a universal [dbt Semantic Layer](/docs/use-dbt-semantic-layer/dbt-sl), you'll need a dbt Cloud [Team or Enterprise](https://www.getdbt.com/pricing/) account. 
```

Reply:
gotcha @matthewshaver great shout! core, dev plan, enterprise, and team plan users can all define their metrics in their tool of choice (cli or ide). however, to query metrics in downstream tools (BI tools) they need to have api access, which is only available to team or enterprise